Radzen.Blazor/Rendering/PopupOrInline.razor

```diff
@@ -0,0 +1,54 @@
+﻿@inherits RadzenComponent
```

Comment:
In my opinion this component is not needed. You can close the popup if Inline is changed in the main component.

Reply:
I forgot that `RenderFragment` exists when I created this component. Removed it now.